src/validation/ValidationSchema.js

```diff
@@ -16,7 +16,7 @@ const validationSchema = yup.object({
     .required('Please enter the scholarship website'),
   amount: yup.object().shape({
     type: yup.mixed().required('Please choose an option above'),
-    minAmount: yup
+    min: yup
       .number()
       .when('type', {
         is: AmountType.Fixed,
```

Comment:
Btw, I think the `.number()` in line 24 might be redundant with the one from line 20

Reply:
I need it in order to chain the `.required().moreThan()`. Without it, it throws an error :/

Reply:
ohhh I see 👍